src/modules/formBuilder/formBuilderUtil.ts

```diff
@@ -82,6 +82,39 @@ export const validComponentsForType = (type: ItemType) => {
       return [];
   }
 };
+export const determineAutofillField = (itemType: ItemType) => {
```

Comment:
Could these be combined into 1 function?

Reply:
They could -- they are currently equivalent -- but they won't remain that way. They have som fields that differ but aren't support yet. I didn't want to give the wrong impression that they were equivalent.